examples/flutter_gallery/lib/demo/cupertino/cupertino_alert_demo.dart

```diff
@@ -92,13 +94,16 @@ class _CupertinoAlertDemoState extends State<CupertinoAlertDemo> {
           CupertinoButton(
             child: const Text('Alert with Title'),
             color: CupertinoColors.activeBlue,
-            padding: const EdgeInsets.symmetric(vertical: 16.0, horizontal: 36.0),
+            padding:
+                const EdgeInsets.symmetric(vertical: 16.0, horizontal: 36.0),
             onPressed: () {
               showDemoDialog<String>(
                 context: context,
                 child: CupertinoAlertDialog(
-                  title: const Text('Allow "Maps" to access your location while you are using the app?'),
-                  content: const Text('Your current location will be displayed on the map and used '
+                  title: const Text(
+                      'Allow "Maps" to access your location while you are using the app?'),
+                  content: const Text(
+                      'Your current location will be displayed on the map and used '
                       'for directions, nearby search results, and estimated travel times.'),
```

Comment:
the original seems clearer. maybe if dartfmt was run with a wider line length?

Reply:
Sounds good. The main friction is that 80 is a hard threshold for the default formatter settings and a suggestion for package:flutter

Reply:
The 2 lines being different now is definitely pretty weird. 

Seems like a general problem where we have 2 states, everything in one line or every parameter on its own line whereas dartfmt has a third state of trying to fit all the parameters together on the second line and end the statement there if possible. But that option looks weird in Flutter with all the nesting.

Reply:
I think I've added a reasonable way to keep Dart in the 2 states for existing flutter code. I've enhanced the fix code to also add new trailing commas to lines that were over 80 characters previously as for these lines where the author thought the invocation fit on one line and so didn't use trailing commas. Please take another look.